libs/chain-signatures/contract/src/primitives/ckd.rs

```diff
@@ -0,0 +1,25 @@
+use near_sdk::{near, AccountId};
+
+// TODO: do we need the Default trait here?
+#[derive(Clone, Debug)]
+#[near(serializers=[json])]
+pub struct CKDRequestArgs {
+    pub app_public_key: near_sdk::PublicKey,
```

Comment:
This could be a `SerializableAffinePoint` instead, making the ckd API more uniform. Let me know which one should be preferable. The good thing about this is that it is a much simpler type, the bad is that it includes edwards curves as well. Using k256_types::PublicKey is also an option, but would require adding all the serialization code for that type as well.

Reply:
what would be your preference?

Reply:
I think I prefer SerializableAffinePoint, will change the code to reflect that if you have nothing against it